src/wayland/generator/interface.cpp

```diff
@@ -22,23 +22,53 @@
 
 #include "method.h"
 
+namespace
+{
+auto matching_keys_to_vector(
+    std::unordered_multimap<std::string, std::string> const& map,
+    std::function<std::string(std::string)> const& name_transform,
+    std::string key)
+{
+    auto range = map.equal_range(key);
+
+    if (range.first == map.end())
+        return std::vector<std::string>{};
```

Comment:
From the documentation (on en.cppreference) wasn't clear to me *what* the value of `range.second` is in the case where the key is not found in the map, only that `range.first` would equal `end()`.

I suppose it would be churlish of a libstdc++ implementation *not* to return `range.first == range.second == map.end()` in that case, though.

Reply:
It really is documented to be end() if there are no elements > key.